src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerActivityExtensions.cs

```diff
@@ -62,10 +61,7 @@ public static JaegerSpan ToJaegerSpan(this Activity activity)
                 Tags = PooledList<JaegerTag>.Create(),
             };
 
-            DictionaryEnumerator<string, object, TagState>.AllocationFreeForEach(
-                activity.TagObjects,
-                ref jaegerTags,
-                ProcessActivityTagRef);
+            activity.EnumerateTagValues(ref jaegerTags);
```

Comment:
@alanwest FYI I updated this for the new extension. It is a slightly easier API to use, and a bit faster because it doesn't hit the concurrent dictionary each time.